README.md

```diff
@@ -1,24 +1,158 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Catalog Of My Things
 
```

Comment:
- [ ] You are using Rspec, the `test` folder is used by another test framework.  Please, kindly consider deleting it if you are not using it.